spec/index.html

```diff
@@ -0,0 +1,546 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>MiniApp Widget Specification</title>
+
+  <link rel="stylesheet" href="local.css" />
+
+  <style>
+    table {
+      border-collapse: collapse;
+    }
+
+    table,
+    th,
+    td {
+      border: 1px solid #666;
+    }
+
+    td {
+      padding: 2px 15px;
+    }
+  </style>
+
+  <script async class="remove" src="https://www.w3.org/Tools/respec/respec-w3c"></script>
+  <script defer src="script.js"></script>
+
+  <script class="remove">
+    var respecConfig = {
+      specStatus: "ED",
+      copyrightStart: "2022",
+      edDraftURI: "https://w3c.github.io/miniapp-widget/sepc/",
+
+      shortName: "miniapp-widget-spec",
+      noRecTrack: true,
+
+      editors: [{
+        name: "Yinli Chen",
+        companyURL: "https://www.mi.com/",
+        company: "Xiaomi",
+        w3cid: 117298
+      }, {
+        name: "Xiaoping Zhao",
+        companyURL: "https://www.mi.com/",
+        company: "Xiaomi",
+        w3cid: 137575
+      }, {
+        name: "Bingqing Zhou",
+        companyURL: "https://www.mi.com/",
+        company: "Xiaomi",
+        w3cid: 136431
+      }, {
+        name: "Canfeng Chen",
+        companyURL: "https://www.mi.com/",
+        company: "Xiaomi",
+        w3cid: 124040
+      }, {
+        name: "Xiangyang Yao",
+        companyURL: "https://www.mi.com/",
+        company: "Xiaomi"
+      }, {
+        name: "Wen Pan",
+        companyURL: "https://www.mi.com/",
+        company: "Xiaomi"
+      }, {
+        name: "Hai Jiang",
+        companyURL: "https://www.mi.com/",
+        company: "Xiaomi"
```

Comment:
Are these three editors members of the MiniApps CG? We need to ensure that every editor/contributor is a member of the community group for IPR reasons.